### PR TITLE
another api endpoint

### PR DESCRIPTION
**Pull request by chriddyp**

### CODE REVIEW

**Comment by chriddyp** [comment]
cc @Kully @cldougl

**Comment by Kully** [comment]
@chriddyp looking at it now

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 - offline embedded plots are now responsive to window resizing when `output_type == "div"` is set in `plotly.offline.iplot()`.
+- Beta: Methods were added that wrap the API endpoint for managing Dash objects on plot.ly. The API interface is under `plotly.api.v2.dash_apps`
```
Comment (Kully) [inline CHANGELOG.md]: I think present tense sounds better here, so `Methods (are added) that wrap the API endpoint ...`.

File: plotly/api/v2/dash_apps.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+def create(body):
+    """Create a dash app item."""
+    url = build_url(RESOURCE)
+    return request('post', url, json=body)
+
+
+def retrieve(fid):
+    """Retrieve a dashboard from Plotly."""
```
Comment (Kully) [inline plotly/api/v2/dash_apps.py]: Is this supposed to say dashboard? Are dash apps just dashboards?

Comment (chriddyp) [inline plotly/api/v2/dash_apps.py]: woops, that's a typo. updated

File: test/vv/lib/python2.7/site-packages/plotly/api/v2/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+from plotly.api.v2 import (dashboards, files, folders, grids, images, plot_schema,
```
Comment (Kully) [inline test/vv/lib/python2.7/site-packages/plotly/api/v2/__init__.py]: What is the point of this file? Is this a file for the purpose of testing out our v2 stuff?

**Comment by cldougl** [comment]
lgtm 💃 
looks like there's some issues w/ the tests related to stage atm

**Comment by Kully** [comment]
💃 from me as well - after tests

File: test/vv/lib/python2.7/site-packages/plotly/api/v2/__init__.py (hunk 3 above)
Comment (chriddyp) [inline test/vv/lib/python2.7/site-packages/plotly/api/v2/__init__.py]: woops didn't meant to commit that

**Pull request merged**